25_exceptions_lab/02.py

```diff
@@ -4,4 +4,24 @@
 
 Alert the user politely if there was any problem opening the file
 """
-
+import os
+import argparse
+
+# Define input arguments
+parser = argparse.ArgumentParser()
+parser.add_argument("file_path", type=str, help="The file path to count lines of.")
+
+# Read input arguments
+args = parser.parse_args()
+
+try:
+    if not os.path.exists(args.file_path):

```

Comment:
No need to check for path existence. If you got the wrong path open will raise an exception. Prefer to add an except block to open